apps/web/app/layout.tsx

```diff
@@ -16,11 +13,7 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body>
-        <TRPCProvider>
-          <Nav />
-          {children}
-          <Footer />
-        </TRPCProvider>
+        <TRPCProvider>{children}</TRPCProvider>
```

Comment:
The Nav and Footer components now won't apply to the full layout, just the files inside (pages)